libfwupdplugin/fu-bios-settings.c

```diff
@@ -555,6 +555,33 @@ fu_bios_settings_get_all(FuBiosSettings *self)
 	return g_ptr_array_ref(self->attrs);
 }
 
+/**
+ * fu_bios_settings_get_pending_values:
```

Comment:
I'm a little apprehensive about the use of the word `pending` in this context.  There is also a `pending_reboot` sysfs file, and I think by using `pending` the two can get mixed up.  How about `unsettled`?

Reply:
Sounds good. I'll replace them :)